lib/ccflay.rb

```diff
@@ -84,3 +84,25 @@ def pure_ruby_hash # :nodoc: see above
     hash
   end
 end
+
+class Sexp
```

Comment:
Is there a reason you re-open again, instead of including `#flatter` in the class definition opened at L38?